app/models/proposal.rb

```diff
@@ -119,7 +115,7 @@ def existing_observation_for(user)
   end
 
   def eligible_observers
-    if observations.count > 0
+    if observations.any?
```

Comment:
afaik this results is different sql and may be impacted by object caching.

`observations.count` will execute a SQL `select count(*) from ....` while `observations.any?` loads the objects (if not already) and counts them.

The former is more accurate and efficient, imo.